umake/frameworks/games.py

```diff
@@ -385,3 +387,70 @@ def post_install(self):
                         exec=self.exec_link_name,
                         comment=self.description,
                         categories="Development;IDE;"))
+
+
+class Godot(umake.frameworks.baseinstaller.BaseInstaller):
+
+    def __init__(self, **kwargs):
+        super().__init__(name="Godot",
+                         description=_("Multi-platform 2D and 3D opensource game engine"),
+                         only_on_archs=['i386', 'amd64'],
+                         download_page="https://godotengine.org/download",
+                         desktop_filename="godot.desktop",
+                         required_files_path=["godot"],
+                         dir_to_decompress_in_tarball="",
+                         icon_filename='icon.png',
+                         **kwargs)
+        self.icon_url = os.path.join("https://raw.githubusercontent.com/godotengine/godot/",
+                                     "master",
+                                     self.icon_filename)
+        self.exec_name = ''
```

Comment:
exec_name is set automagically, don't override it (it's set as the first element in the required_files_path)

Reply:
Executable name again. I need to this name to rename Godot executable `Godot_v2.1.4-stable_x11.64` after install.